cclib/parser/daltonparser.py

```diff
@@ -82,10 +82,12 @@ def extract(self, inputfile, line):
         # Extract the version number and optionally the Git revision
         # number.
         if line[4:30] == "This is output from DALTON":
```

Comment:
Can you add some examples that the regex is trying to match?

Reply:
Let me know if the comment block is enough.